### PR TITLE
Markers do not disappear after marker styles are renamed

### PR DESCRIPTION
**Pull request by MaksymShokin**
@sofiiakvasnevska

## Issue
Fliplet/fliplet-studio#5407

## Description
When the user changes the marker style name, it will be updated in data source.

## Screenshots/screencasts
interactive-fix

## Backward compatibility
This change is fully backward compatible.

### CODE REVIEW

File: js/interface/add-markers.js [hunk 1]
```diff
@@ -335,8 +337,53 @@ Fliplet.InteractiveMap.component('add-markers', {
...
+        Fliplet.DataSources.connect(this.dataSourceId).then(connection => {
+          this.dataSourceConnection = connection
+
+          connection.find().then(records => {
+            if (!records.length) {
+              return
+            }
+
+            records.forEach(elem => {
+              var matchedStyleName = _.find(this.styleNames, function(style) {
```
Comment (squallstar) [inline js/interface/add-markers.js]: `var` is not allowed when using code that gets transpiled

File: js/interface/add-markers.js [hunk 2]
```diff
@@ -335,8 +337,53 @@ Fliplet.InteractiveMap.component('add-markers', {
...
+                return style.oldStyleName === elem.data['Marker style'];
+              });
+
+              if (matchedStyleName) {
+                elem.data['Marker style'] = matchedStyleName.newStyleName
+              }
+            })
+
+            let columns = _.keys(records[0].data)
```
Comment (squallstar) [inline js/interface/add-markers.js]: why not `const`?

File: templates/interface/add-markers.interface.hbs [hunk 3]
```diff
@@ -37,7 +37,7 @@
           <template v-else>
             You are using the data source called <strong>\{{markersDataSource.name}}</strong>.<br>
             <span>
-              <a href="#" @click.prevent="editDataSource">Bulk upload data</a> | 
+              <a href="#" @click.prevent="editDataSource">Bulk upload data</a> |
```
Comment (squallstar) [inline templates/interface/add-markers.interface.hbs]: you sure this change hasn't removed a required space? If so, use `&nbsp;`

Comment (MaksymShokin) [reply templates/interface/add-markers.interface.hbs]: Everything works as before.

File: js/interface/add-markers.js (hunk 1 above)
Comment (MaksymShokin) [reply js/interface/add-markers.js]: fixed

File: js/interface/add-markers.js (hunk 2 above)
Comment (MaksymShokin) [reply js/interface/add-markers.js]: fixed

**Pull request merged**